src/main/java/Line.java

```diff
@@ -0,0 +1,57 @@
+public class Line {
+    private int fromLine;
+    private int toLine;
+    private int t;
+    private int[][] rows;
+
+    public Line(int fromLine, int toLine, int currentRow, int[][] rows) {
+        sort(fromLine, toLine);
+        this.rows = rows;
+
+        if (validateFromLine(rows) && currentRow >= 0 && validateCurrentRow(currentRow, rows)) {
+            this.rows[currentRow][this.fromLine] = 1;
+        }
+        if (validateToLine(rows) && currentRow >= 0 && validateCurrentRow(currentRow, rows)) {
+            this.rows[currentRow][this.toLine] = - 1;
+        }
+    }
+
+    private static boolean validateCurrentRow(int currentRow, int[][] rows) {
+        if(currentRow < rows.length){
+            return true;
+        }
+        else{
+            throw new IllegalArgumentException(ExceptionMessage.INVALID_LINE_POSITION.getMessage());
+        }
+    }
```

Comment:
else를 안쓰고 throw를 바로 써도 좋아요.

무조건 이렇게 써야하는거는 아니지만, 이런식으로 쓰면 편한 경우가 자주 있어서 알아두시면 좋아요. 👍 
```suggestion
    private static boolean validateCurrentRow(int currentRow, int[][] rows) {
        if(currentRow < rows.length){
            return true;
        }
        throw new IllegalArgumentException(ExceptionMessage.INVALID_LINE_POSITION.getMessage());
    }
```